src/python/WMCore/WMSpec/WMWorkloadTools.py

```diff
@@ -153,13 +127,18 @@ def _validateArgumentOptions(arguments, argumentDefinition, optionKey):
 
 
 def _validateInputDataset(arguments):
+    
     inputdataset = arguments.get("InputDataset", None)
     dbsURL = arguments.get("DbsUrl", None)
     if inputdataset != None and dbsURL != None:
-        result = _verifyDBSCall(dbsURL, "datasets?&dataset_access_type=*&dataset=%s" % inputdataset)
-        if len(result) == 0:
-            msg = "Inputdataset %s doesn't exist on %s" % (inputdataset, dbsURL)
-            raise WMSpecFactoryException(msg)
+        #import DBS3Reader here, since Runtime code import this module and worker node doesn't have dbs3 client 
+        from WMCore.Services.DBS.DBS3Reader import DBS3Reader
+        from WMCore.Services.DBS.DBSErrors import DBSReaderError
+        try:
```

Comment:
I was planning to move the functions in different module and not include the package packed but this module is not directly accessed in the WN but through WMSpec. So I imported when DBSService with in the module. I could create the function for import and call that function with in other function but one method need 2 import and the other need just one.  I know it is ugly but couldn't improve much with the simplest change.